src/main/java/core/basesyntax/services/LineParser.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.services;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+
+public class LineParser {
+    public static final int CORRECT_ARG_AMOUNT = 4;
```

Comment:
if we don't use these variables outside of the class, i suggest we make them private